tests/main/interfaces-posix-mq/task.yaml

```diff
@@ -0,0 +1,78 @@
+summary: Ensure that the posix-mq interface works
+
+details: |
+    The posix-mq interface allows access to the POSIX message queues. This test
+    explores a number of features related to distinct sub-permissions implemented
+    in snapd - read, write, create and delete.
+
+systems:
+    # Too old to support this feature entirely.
+    - -ubuntu-14.04-64
+    - -debian-11-64
+    # Distribution AppArmor parser does not support mqueue mediation.
+    # AppArmor parser does not re-execute despite SNAPD_APPARMOR_REEXEC=1 in /usr/lib/snapd/info.
+    - -opensuse-15.6-64
+    # Affected by https://gitlab.com/apparmor/apparmor/-/issues/492
+    - -ubuntu-16.04-64
+    - -ubuntu-18.04-64
+    - -ubuntu-core-16-64
+    - -ubuntu-core-18-64
+    - -ubuntu-core-20-64
+    - -debian-12-64
+    - -debian-13-64
+    - -debian-sid-64
+    - -arch-linux-64
+    - -opensuse-tumbleweed-64
+
+prepare: |
+    # Source code is at https://github.com/canonical/test-snapd-posix-mq
```

Comment:
why isn't this snap under tests/lib/snaps/store ?

Reply:
Ah, I didn't know this existed. Shall I just import the history? I kind of like the fact that the snap can be build from the store and auto-published.

Reply:
let's do that in a followup